### ISSUE TITLE
[Keyboard navigation - ADF - Azure data factory] Keyboard focus is not visible on Azure data factory control at left panel.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3841]
**User Experience:**
User will find difficulty to see any control if keyboard focus is not visible on any interactive control as they won't be able to take an action on such control and also they will surely miss the functionality and will proceed further.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Note:** User credentials should NOT be included in the bug.

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it. 

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate to visual icon Add data factory control at left panel.
- Verify focus visibility on visual icon Add data factory control.

**Actual Result:**
Keyboard focus is not visible on Azure data factory control at left panel and due to this, visual impaired user would difficulty to see the focus visibility on that control and also he/she won't be perform functionality due to no visibility.

**Expected Result:**
Keyboard focus should be visible on Azure data factory control at left panel so that visual impaired user could easily see the keyboard focus on that control to proceed further.

[MAS2.4.7_Azure Data factory.zip](https://github.com/microsoft/AzureStorageExplorer/files/5642905/MAS2.4.7_Azure.Data.factory.zip)

**Comment by fhljys** [comment]
There is a blue indicator when it is in focus state. 
![image](https://user-images.githubusercontent.com/18674958/122521594-98381600-d047-11eb-8e7d-ca06eae1994d.png)
Is it acceptable?

**Comment by jinglouMSFT** [comment]
This is how VS Code shows the focus except they use a white bar instead of a blue bar.
